docs/developer-docs/setup/pocket-ic.md

```diff
@@ -0,0 +1,278 @@
+
+```
+xattr -dr com.apple.quarantine pocket-ic
+```
+
+The PocketIC binary can be left in your working directory or you can specify the path to the binary by setting the `POCKET_IC_BIN` environmental variable. 
+
+Then, run the following command to install PocketIC in your Python virtual environment:
+
+```
+pip3 install pocket-ic
+```
+
+To import PocketIC into your Python code, use the line:
+
+```python
+from pocket_ic import PocketIC
+```
+
+### Example: create a canister with cycles
+
+```python
+from pocket_ic import PocketIC
+
+pic = PocketIC()
+canister_id = pic.create_canister()
+pic.add_cycles(canister_id, 2_000_000_000_000)
+```
+
+
+### Make canister calls
+
+```python
+response = pic.update_call(canister_id, method="greeting", ...)
+assert(response == 'Hello, PocketIC!')
+```
+
+### Make a call directly with a canister object
+
+```python
+my_canister = pic.create_and_install_canister_with_candid(...)
+```
+
+### Call your canister methods using the native Python syntax
+
+```python
+response = my_canister.greeting()
+assert(response == 'Hello, PocketIC!')
+```
+
+### Example: Single canister testing on a single subnet
+
+The following is a simple, but complete testing example with a counter canister:
+
+```python
+import sys
+import os
+import unittest
+import ic
+
+# The example needs to have the module in its sys path, so we traverse
+# up until we find the pocket_ic package.
+script_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(os.path.dirname(script_dir)))
+
+from pocket_ic import PocketIC
+
+
+class CounterCanisterTests(unittest.TestCase):
+    def test_counter_canister(self):
+        pic = PocketIC()
+        canister_id = pic.create_canister()
+        pic.add_cycles(canister_id, 1_000_000_000_000_000_000)
+
+        with open(os.path.join(script_dir, "counter.wasm"), "rb") as wasm_file:
+            wasm_module = wasm_file.read()
+        pic.install_code(canister_id, bytes(wasm_module), [])
+
+        self.assertEqual(
+            pic.update_call(canister_id, "read", ic.encode([])),
+            [0, 0, 0, 0],
+        )
+        self.assertEqual(
+            pic.update_call(canister_id, "write", ic.encode([])),
+            [1, 0, 0, 0],
+        )
+        self.assertEqual(
+            pic.update_call(canister_id, "write", ic.encode([])),
+            [2, 0, 0, 0],
+        )
+        self.assertEqual(
+            pic.update_call(canister_id, "read", ic.encode([])),
+            [2, 0, 0, 0],
+        )
+
+
+if __name__ == "__main__":
+    unittest.main()
+```
+
+To see more examples and run them locally, clone the [PocketIC Python repo](https://github.com/dfinity/pocketic-py/tree/main), then run the following command, replacing the example name with the example you'd like to run. For example, to run the `counter_canister` example, use the command:
+
+```
+run python3 examples/counter_canister/counter_canister_test.py
+```
+
+## Resources
+
```

Comment:
I think it'd be great to mention the main PocketIC repo as the first resource here: https://github.com/dfinity/pocketic